### PR TITLE
feat: Refactorized SettingsVC

### PR DESCRIPTION
**Pull request by jasonskycore**
feat: Made some changes to TableViewCell in SettingsVC
feat: Added PushNotificationVC

### CODE REVIEW

File: Sounds Like A Plan/Features/Profile-old/SwitchTableViewCell.swift [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+//  Hayo
+//
+//  Created by Jason Fernando on 02/12/22.
+//
+
+import UIKit
+
+class SwitchTableViewCell: UITableViewCell {
+    
+    static let identifier = "SwitchTableViewCell"
```
Comment (gremlinflat) [inline Sounds Like A Plan/Features/Profile-old/SwitchTableViewCell.swift:12]: identifier should be lowercased

File: Sounds Like A Plan/Features/Profile-old/SettingsViewController.swift [hunk 2]
```diff
@@ -1,158 +1,185 @@
...
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        view.backgroundColor = .systemBackground
+        
+        title = "Settings"
+        //FIXME: navbar font di profile ikut jadi 17 sizenya
+        self.navigationController?.navigationBar.titleTextAttributes = [NSAttributedString.Key.font: UIFont.systemFont(ofSize: 17, weight: .bold)]
+        
+        let rightButton = UIBarButtonItem(barButtonSystemItem: .done, target: self, action: #selector(onDoneButtonTapped))
```
Comment (gremlinflat) [inline Sounds Like A Plan/Features/Profile-old/SettingsViewController.swift:29]: did___tapped

Comment (jasonskycore) [reply Sounds Like A Plan/Features/Profile-old/SettingsViewController.swift:29]: siap noted

File: Sounds Like A Plan/Features/Profile-old/SwitchTableViewCell.swift (hunk 1 above)
Comment (jasonskycore) [reply Sounds Like A Plan/Features/Profile-old/SwitchTableViewCell.swift:12]: siap noted🙏🙏

File: Sounds Like A Plan/Features/Profile-old/SettingsViewController.swift [hunk 3]
```diff
@@ -1,158 +1,185 @@
...
+        configure()
+        setupTableView()
+        setupTableViewFooter()
+    }
+    
+    @objc func onDoneButtonTapped() {
+        self.navigationController?.popViewController(animated: true)
+    }
+    
+    func configure() {
```
Comment (gremlinflat) [inline Sounds Like A Plan/Features/Profile-old/SettingsViewController.swift:43]: why dont you append it on line 19? 
e.g

let settings: [Settings] = [
     Setting(type: "PREFERENCES", content: ["Push notifications"]),
     Setting(type: "ACCOUNT SETTINGS", content: ["Personal Information", "Privacy"]),
     .......
]

File: Sounds Like A Plan/Features/Profile-old/PushNotificationViewController.swift [hunk 4]
```diff
@@ -1,20 +1,135 @@
...
+        
+        configure()
+        setupTableView()
+    }
+    
+    @objc func onDoneButtonTapped() {
+        self.navigationController?.popViewController(animated: true)
+    }
+    
+    func configure() {
```
Comment (gremlinflat) [inline Sounds Like A Plan/Features/Profile-old/PushNotificationViewController.swift:32]: sama kyk diatas

File: Sounds Like A Plan/Features/Profile-new/ProfileViewController.swift [hunk 5]
```diff
@@ -192,7 +192,8 @@ class ProfileViewController: UIViewController {
     
     //FIXME: - Belum ditambahkan pencet Setting Button
     @objc func onSettingButtonTapped() {
```
Comment (gremlinflat) [inline Sounds Like A Plan/Features/Profile-new/ProfileViewController.swift:194]: did__tapped

**Review by gremlinflat — CHANGES_REQUESTED** [review]

File: Sounds Like A Plan/Features/Profile-old/SettingsViewController.swift (hunk 3 above)
Comment (jasonskycore) [reply Sounds Like A Plan/Features/Profile-old/SettingsViewController.swift:43]: noted sir, will make it tidier 🙏

File: Sounds Like A Plan/Features/Profile-old/PushNotificationViewController.swift (hunk 4 above)
Comment (jasonskycore) [reply Sounds Like A Plan/Features/Profile-old/PushNotificationViewController.swift:32]: ok

**Review by gremlinflat — APPROVED** [review]
LGTM

**Pull request merged**